src/styles/blocks/price.scss

```diff
@@ -0,0 +1,53 @@
+.price {
+  &__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    align-items: center;
+    margin-block: 24px 16px;
+    margin-inline: $margin-inline;
```

Comment:
Ensure that the SCSS variable `$margin-inline` is defined in your stylesheets. If not, you need to define it or replace it with a valid margin value.